spec/user_repository_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'user'
+require 'user_repository'
+
+def reset_users_table
+    seed_sql = File.read('spec/seeds/seeds_users.sql')
+    connection = PG.connect({ host: '127.0.0.1', dbname: 'chitter_database' })
+    connection.exec(seed_sql)
+  end
+  
+  describe UserRepository do
+    before(:each) do 
+      reset_users_table
+    end
+
+    it 'checks if the email is in use' do
+        repo = UserRepository.new
+        found_user = repo.find('mike@strangerthings.com')
+
+        if found_user.nil?
```

Comment:
Instead of putting the condition in the test, you may try add the logic for checking existing user in the repository.